src/ripple/nodestore/Database.h

```diff
@@ -322,6 +313,12 @@ class Database : public Stoppable
     virtual void
     for_each(std::function<void(std::shared_ptr<NodeObject>)> f) = 0;
 
+    /** Retrieve backend read and write stats.
+        The return pointer is treated like an std::optional.
```

Comment:
I'd explicitly say it's treated like a "std::optional to a reference". The reason we're using a pointer is because of the reference, otherwise we would just use a regular `optional`.